.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+dist: trusty
+language: python
+python:
+  - "3.5"
+
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
Are you sure those are packaged together on Ubuntu? That would make the clang-format package redundant.